packages/rest-typings/src/v1/misc.ts

```diff
@@ -206,14 +206,14 @@ export type MiscEndpoints = {
 	'/v1/pw.getPolicy': {
 		GET: () => {
 			enabled: boolean;
-			policy: [name: string, options?: Record<string, unknown>][];
+			policy: [name: string, value?: Record<string, number>][];
 		};
 	};
 
 	'/v1/pw.getPolicyReset': {
 		GET: (params: PwGetPolicyReset) => {
 			enabled: boolean;
-			policy: [name: string, options?: Record<string, unknown>][];
+			policy: [name: string, value?: Record<string, number>][];
 		};
 	};
 
```

Comment:
I don't think that the token it'll be necessary since the auth is not required... maybe it worth remove one of the endpoints (check with Backend)

Reply:
I can't remove the endpoint, it has to be deprecated first and removed on future releases.